src/components/Login/Login.jsx

```diff
@@ -39,6 +42,11 @@ export const Login = () => {
       })
       .catch((err) => console.log(err));
   };
+
+  setTimeout(() => {
+    setErrors("");
+  }, 4000);
```

Comment:
Why are we using setTimeout outside the useEffect hook?
Use it inside hook (useEffect) with some dependency(s).